k8s/client.go

```diff
@@ -929,15 +929,15 @@ func (c *Client) ListCiliumPodIPPools(ctx context.Context, opts metav1.ListOptio
 	return c.CiliumClientset.CiliumV2alpha1().CiliumPodIPPools().List(ctx, opts)
 }
 
-func (c *Client) GetLogs(ctx context.Context, namespace, name, container string, sinceTime time.Time, limitBytes int64, previous bool) (string, error) {
+func (c *Client) GetLogs(ctx context.Context, namespace, name, container string, sinceTime time.Time, limitBytes int64, previous, timestamps bool) (string, error) {
```

Comment:
I understand if this might be out of scope for this pr but it seems that this call might as well take in PodLogOptions?

Reply:
Sounds reasonable to me. I've updated it to take `PodLogOptions` instead of adding the extra parameter.